requirements/dev.txt

```diff
@@ -10,6 +10,10 @@ pytest>=3.0.7
 mock>=2.0.0
 parameterized>=0.6.1
 
+# Requirements for integration tests
+certifi>=2020.4.5.1
+urllib3
```

Comment:
Wait, we already have a dependency on requests but add urllib3? We should just be able to use requests for calls.

Reply:
Updated the code to use `requests`